src/RA_Dlg_AchEditor.cpp

```diff
@@ -333,7 +336,7 @@ long _stdcall EditProc(HWND hwnd, UINT nMsg, WPARAM wParam, LPARAM lParam)
             ZeroMemory(&lvDispinfo, sizeof(LV_DISPINFO));
             lvDispinfo.hdr.hwndFrom = hwnd;
             lvDispinfo.hdr.idFrom = GetDlgCtrlID(hwnd);
-            lvDispinfo.hdr.code = LVN_ENDLABELEDIT;
+            lvDispinfo.hdr.code = std::numeric_limits<UINT>::max();
```

Comment:
Don't know what microsoft was thinking giving negative values to unsigned numbers. Negative anything will just give the largest number that unsigned type can hold.